Automattic/jetpack
# SSO: Persist user-new locale and custom message  user-new.php form field data

Related to https://github.com/Automattic/jetpack/pull/35277#issuecomment-1931904940




## Proposed changes:

* Persists user-new.php `locale` and `custom message` form field data
* Updates new-user scripts to load using Jetpack Assets.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. On an Atomic, JN, or whatever non-simple site, install Jetpack.
2. Make sure SSO is on, it should be on by default.
3. Sync this PR to your site.
4. Navigate to /wp-admin/user-new.php and submit a new user with an incorrect email.
5. Verify the `locale` and `custom message` form fields persist.
